@nadavMiz attaching the warning:
```
Jun-30 10:23:04.557 [Endpoint/12] [ERROR] CONSOLE:: Warning! Postgres only supports 63 characters for query names.
Jun-30 10:23:04.557 [Endpoint/12] [ERROR] CONSOLE:: You supplied find_object_with_mapping_by_key:objectmds:objectparts:datachunks:datablocks (75)
Jun-30 10:23:04.557 [Endpoint/12] [ERROR] CONSOLE:: This can cause conflicts and silent errors executing queries
```

If you can edit the PR description from 68 to 63 (typo)

--- a/src/server/object_services/md_store.js
+++ b/src/server/object_services/md_store.js
@@ -34,6 +34,7 @@ const sql_and_conditions = (...conditions) => conditions.filter(Boolean).join(' 
 class MDStore {
 
     constructor(test_suffix = '') {
+        this._test_suffix = test_suffix;
         this._postgres_pool = 'md';
 
         this._objects = db_client.instance().define_collection({
@@ -2090,9 +2091,7 @@ class MDStore {
         const values = [String(obj_id), start_gte, start_lt, end_gt];
         const res = await db_client.instance().executeSQL(query, values, {
             preferred_pool: this._postgres_pool,
-            query_name: config.DB_PREPARED_STATEMENTS_ENABLED ?
-                `find_parts_chunks_blocks_by_range:${this._parts.name}:${this._chunks.name}:${this._blocks.name}` :
-                undefined,
+            query_name: config.DB_PREPARED_STATEMENTS_ENABLED ? `fpcbb${this._test_suffix}` : undefined,
         });
         return _parse_mapping(res.rows[0]?.mapping, sorter);
     }
@@ -2156,9 +2155,7 @@ class MDStore {
         const values = [`${bucket_id}`, key, max_parts];
         const res = await db_client.instance().executeSQL(query, values, {
             preferred_pool: this._postgres_pool,
-            query_name: config.DB_PREPARED_STATEMENTS_ENABLED ?
-                `find_object_with_mapping_by_key:${this._objects.name}:${this._parts.name}:${this._chunks.name}:${this._blocks.name}` :
-                undefined,
+            query_name: config.DB_PREPARED_STATEMENTS_ENABLED ? `fowmbk${this._test_suffix}` : undefined,
         });
         if (!res.rows.length) return null;
         const row = res.rows[0];

